loadbalancer/etcd/publisher_test.go

```diff
@@ -0,0 +1,107 @@
+package etcd
```

Comment:
It should be fine, if you follow standard Go forking practice, by adding your repo as a remote to the canonical checkout, and working in the standard path.